src/components/pages/Profile/RenderProfileContainer.js

```diff
@@ -66,6 +66,21 @@ const RenderProfileContainer = props => {
             <Card type="inner" title="Bio" extra={<EditOutlined />}>
               {userData.bio || 'Profile bio will go here.'}
             </Card>
+            <Card type="inner" title="Full Name" extra={<EditOutlined />}>
+              {userData.full_name || 'Full Name will go here.'}
```

Comment:
Why require the backend to send the full name?  We do often want to get a full name on the front end. Maybe we should use a utility that joins them in the front end.